blocks/v2-video/v2-video.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-class-pattern */
+.v2-video-wrapper {
+  --video-inset: inset(32px 48px round 8px);
+
+  padding: 0 !important;
+  color: var(--c-white);
+  margin: 0 !important;
+  width: 100% !important;
+}
+
+.v2-video {
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  clip-path: inset(24px 24px round 8px);
+  transition: clip-path var(--duration-large) ease-in-out;
+}
+
+.v2-video.v2-video--full-width {
+  clip-path: inset(0 0 round 0);
+}
+
+.v2-video .v2-video__video {
+  position: absolute;
+  top: 0;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+  max-width: 100%;
+  width: 100%;
+}
+
+.v2-video .v2-video__content-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  background: linear-gradient(0deg, rgb(0 0 0 / 50%) 0%, rgb(0 0 0 / 0%) 100%);
+  width: 100%;
+  position: relative;
+}
+
+.v2-video .v2-video__content {
+  margin: 48px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  font-family: var(--font-family-body);
+  font-size: var(--f-body-font-size);
+  font-style: normal;
+  line-height: var(--f-body-line-height);
+  letter-spacing: var(--f-body-letter-spacing);
+  text-align: center;
+}
+
+.v2-video .v2-video__content * {
+  margin: 0;
+}
+
+.v2-video .v2-video__content p {
+  font-family: var(--font-family-body);
+  font-style: normal;
+}
+
+.v2-video .v2-video__heading {
+  font-family: var(--font-family-heading);
+  font-size: var(--f-heading-5-font-size);
+  font-style: normal;
+  line-height: var(--f-heading-5-line-height);
+}
+
+.v2-video .v2-video__button.button.primary {
+  font-size: var(--f-button-font-size);
+  line-height: 100%;
+  letter-spacing: 1.25px;
+  text-transform: uppercase;
+  padding: 9px 0;
+  background: transparent;
+  color: var(--c-white);
+  border: 0;
+  width: auto;
+  display: inline-flex;
+}
+
+@media (min-width: 744px) {
+  .v2-video {
```

Comment:
This is not necessary to be reassigned. You can change the variable in the breakpoint.

`:root { --video-inset: new value;}`

Reply:
fixed